.github/workflows/build-and-deploy.yml

```diff
@@ -203,10 +203,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tests: [ unit_models, unit_services, unit_forms, unit_components, unit_other, integration_auth, integration_find, integration_publish, integration_support, integration_other ]
+        # tests: [ unit_models, unit_services, unit_forms, unit_components, unit_other, integration_auth, integration_find, integration_publish, integration_support, integration_other ]
+        tests: [ unit_services, unit_forms, unit_components, unit_other, integration_auth, integration_find, integration_publish, integration_support, integration_other ]
         include:
-          - tests: unit_models
-            include-pattern: spec/models/.*_spec.rb
+          # - tests: unit_models
+          #   include-pattern: spec/models/.*_spec.rb
```

Comment:
Leftovers?

Reply:
Yea, I'll comment out when all out testing is done